### PR TITLE
Add Appian Icon

### PR DESCRIPTION
**Pull request by mRtyu**
Appian

**Issue:** #6462 
**Alexa rank:** 55,724
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

I downloaded this "appian" SVG from the favicon of their website, followed the procedure written in contributing guidelines, and passed the `lint`.

### CODE REVIEW

**Comment by mRtyu** [comment]
@sachinraja in brand guidelines given below, symbol can be used to represent Appian.

https://assets.appian.com/uploads/assets/Appian_BrandGuidelines_Newsroom.pdf

File: icons/appian.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Appian</title><path d="M19.7 6.1c0-4.4-1.9-6.1-6.1-6.1H7.3v4.1h4.6c1.8 0 2.8 0 2.8 1.6v2.4h-4.4C6 8 4.3 9.8 4.2 14.2v3.7c0 4.1 1.7 5.9 5.4 6.1l3.9-4.1h-1.4c-1.8 0-2.8 0-2.8-1.6v-4.5c0-1.6 1-1.6 2.8-1.6s2.8 0 2.8 1.6V24h4.9V6.1z"/></svg>
```
[the remark this answers is not in the document]
Comment (sachinraja) [reply icons/appian.svg:1]: you can use this path:
```
M19.646 6.117C19.538 1.763 17.883 0 13.636 0H7.34v4.066h4.57c1.799 0 2.807 0 2.807 1.655v2.375c-.828 0-2.88-.036-4.426-.036-4.246 0-5.83 1.727-5.937 6.117v3.742c.108 4.102 1.51 5.865 5.253 6.081l3.85-4.066c-.397.036-.864.036-1.44.036-1.798 0-2.806 0-2.806-1.655v-4.57c0-1.655 1.007-1.655 2.806-1.655 1.908 0 2.807 0 2.807 1.655v10.22h4.821z
```

**Comment by mRtyu** [comment]
@sachinraja i have made the requested changes, please verify.

**Comment by sachinraja** [comment]
Can you also add `https://assets.appian.com/uploads/assets/Appian_BrandGuidelines_Newsroom.pdf` as `"guidelines"`

**Comment by mRtyu** [comment]
> Can you also add `https://assets.appian.com/uploads/assets/Appian_BrandGuidelines_Newsroom.pdf` as `"guidelines"`

done.

**Review by sachinraja — APPROVED** [review]

**Pull request merged**